Source/Scene/Model.js

```diff
@@ -3819,6 +3831,10 @@ function createCommand(model, gltfNode, runtimeNode, context, scene3DOnly) {
       count
     );
 
+    if (primitive.mode === PrimitiveType.POINTS) {
```

Comment:
@lilleyse oh just a question, do we have any data for testing points?

Reply:
No, we don't have any glTF's with points yet. I don't think it needs to hold up this PR, but it's definitely something we need to do once we unify `PointCloud3DTileContent` and `Model`.